app/src/js/components/Granules/overview.js

```diff
@@ -150,7 +150,7 @@ class GranulesOverview extends React.Component {
     };
 
     if (granuleIds.length > 0) {
-      requestBody.granuleIds = granuleIds;
+      requestBody.granuleId = granuleIds;
```

Comment:
line 142 doesn't seem to be correct: granuleIds = [granuleIdFilter];
I'm confused with using granuleIds here.

Reply:
Yeah I think the intention is to deal with passing a string vs. an array. However, since the api handles that anyway, do you think we should omit all of that and do something like this instead?

```
const requestBody = {
  reportName: listName,
  reportType: 'Granule Inventory',
  status,
  collectionId,
  granuleId: granuleIdFilter || selected
};
```

Reply:
Yup.  That's good.

Reply:
Cool, made that change

Reply:
(chiming in because I worked on this) the only reason I didn't want to do `granuleId: granuleIdFilter || selected` was so that the API could always assume this property was an array, like Danielle said. If we don't do it that way it would need to accept either a string in cases where it's just one from the filter or an array in cases where we have multiple checked. 

A user could also just have one selected in the list, in which case you would have an array of one value.

Reply:
granuleIdFilter is not granuleId, so we have to distinguish string and array.  I suggest to add a comment to state that's actually a `search` infix.

Reply:
Alright, spoke with Nate and we agreed that this makes sense given the API can handle strings and arrays. Also added a comment as suggested to clarify what's being passed to that param.